### DISCUSSION TITLE
Can't import SASA

### DISCUSSION THREAD

**Discussion by manajit-das** [rdkit/rdkit#4579] [General] [answered]
I have installed rdkit (version: 2021.03.1), It is working fine, but it is showing the following error "ImportError: cannot import name 'rdFreeSASA' from 'rdkit.Chem'" 
Thanks for your help.

**Comment by ptosco** [comment]
@manajit-das If you have built your own copy of RDKit, you need to enable support for FreeSASA, which is off by default, by adding to your `cmake`  command the relevant flag `-DRDK_BUILD_FREESASA_SUPPORT=ON`. The RDKit version distributed on `conda-forge` includes FreeSASA support.

**Reply by manajit-das** [reply]
Thanks for your answer. I am new to python and cheminformatics. I didn't exactly get what you have mentioned. I didn't make my own copy of RDKit, I just installed using conda. Can the problem be solved if I reinstall the RDkit version distributed on conda-forge?

**Reply by Acario** [reply]
> @manajit-das If you have built your own copy of RDKit, you need to enable support for FreeSASA, which is off by default, by adding to your `cmake` command the relevant flag `-DRDK_BUILD_FREESASA_SUPPORT=ON`. The RDKit version distributed on `conda-forge` includes FreeSASA support.

I have installed the conda-forge version of RDKit and FreeSASA was not installed. Does one still have to go through the cmake build to get FreeSASA to work even with the conda-forge version?

**Reply by greglandrum** [reply]
You don't tell us which operating system you are using, but FreeSASA is not supported on windows, could that be the problem?

**Reply by manajit-das** [reply]
Thanks @greglandrum Yeah, I am using windows. Hence is the problem. Thanks for clarifying.

**Comment by bp-kelley** [comment, marked as answer]
I'll make this an issue and see if the newer releases of FreeSASA perhaps
solve this.

**Reply by manajit-das** [reply]
Thank you. Hope it will be available in the newer releases.
